assets/js/ModuleCategory.js

```diff
@@ -35,6 +36,11 @@ export default class ModuleCategory extends React.Component {
     this.props.setOpenCategory(newCollapsed ? null : this.props.name); // tell parent, so it can close other cats
   }
 
+  collapseAll() {
+    this.setState({collapsed: true});
```

Comment:
Why do we need to both set our state and the parent's state?

Reply:
We don't! Setting the parent is sufficient, removed the setState from this both collapseAll() and toggleCollapse()

Reply:
Is it actually true that display:none is worse for performance than removing and then re-adding things to the DOM?